# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libcap/libcap-1.10-alpha.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Pulled from RedHat SRPM to (mostly) fix samba (3.0.x) compilation on
non-x86 systems. Most of this patches are bare hacks - but seem to be
commonly accepted ... :-(

  - Rene Rebe <rene.rebe@gmx.net>

--- libcap-1.10/libcap/_makenames.c.foo	Fri Nov  9 16:38:45 2001
+++ libcap-1.10/libcap/_makenames.c	Fri Nov  9 16:39:30 2001
@@ -9,7 +9,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
-#include <linux/capability.h>
+#include "include/sys/capability.h"
 
 /*
  * #include 'sed' generated array
